### ISSUE TITLE
Missing Icon in Chilisleep Dock Pro Integration

### ISSUE THREAD

**Issue by rsampayo** [rsampayo/sleepme_thermostat#2]
### Issue Summary:
The Chilisleep Dock Pro integration is displaying a broken icon in Home Assistant, indicating that the icon file is missing or not correctly linked.

### Steps to Reproduce:
1. Install the Chilisleep Dock Pro integration.
2. Set up the integration in Home Assistant.
3. Observe that the entity shows a broken icon or no icon at all.

### Expected Behavior:
The custom icon should be displayed as specified in the integration’s configuration.

### Actual Behavior:
A broken or missing icon is displayed instead.

### Additional Information:
- The icon was intended to be linked via the `manifest.json` file, but it appears that the file path might be incorrect or the icon file is missing.
- No errors are logged in Home Assistant.

**Comment by mikesalz** [comment]
@rsampayo I cloned the Brands repo and created a new local branch.  I then created the icons and logos and added them to a new folder.  But I'm new to adding/modifying HA integrations. 

I first tried git push and got this:

```
$ git push
fatal: The current branch sleepme_thermostat has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin sleepme_thermostat

To have this happen automatically for branches without a tracking
upstream, see 'push.autoSetupRemote' in 'git help config'.

```

I then tried what the error message recommended and got this:
```
$ git push --set-upstream origin sleepme_thermostat
remote: Permission to home-assistant/brands.git denied to mikesalz.
fatal: unable to access 'https://github.com/home-assistant/brands.git/': The requested URL returned error: 403
```
Any thoughts?

**Comment by rsampayo** [comment]
I’m pretty new to github myself, but I think what you might need to do is fork the repository, which creates your own version where you can make changes. After you’ve added your icons, you’d push those changes to your fork. Then, you can create a pull request to ask the maintainers to merge your updates into the main project. This should let you contribute even though you don’t have direct access to the original repository.

**Comment by mikesalz** [comment]
Yeah, that makes perfect sense.  I will give that a try later and let you know how it goes.

**Comment by mikesalz** [comment]
FYI

https://github.com/home-assistant/brands/pull/5775

**Comment by rsampayo** [comment]
Great job!

**Comment by mikesalz** [comment]
Thanks!  Assuming I did everything correctly, I THINK the icons and logos will just show up as soon as the PR is merged.  Fingers crossed!

**Comment by rsampayo** [comment]
I just pushed a new version (forgot to change the version number). Its about issue https://github.com/rsampayo/sleepme_thermostat/issues/6

**Comment by rsampayo** [comment]
@mikesalz I just saw Frenck approved your merge 👏. Can you see the icons in your integration?
